app/components/assets/actions-dropdown.tsx

```diff
@@ -28,75 +28,103 @@ const ConditionalActionsDropdown = ({ asset }: Props) => {
   const [open, setOpen] = useState(defaultOpen);
 
   return (
-    <DropdownMenu modal={true} defaultOpen={open}>
+    <DropdownMenu
+      modal={true}
+      onOpenChange={(open) => setOpen(open)}
+      open={open}
+    >
       <DropdownMenuTrigger className="asset-actions">
         <Button variant="secondary" to="#" data-test-id="assetActionsButton">
           <span className="flex items-center gap-2">
             Actions <ChevronRight className="chev" />
           </span>
         </Button>
       </DropdownMenuTrigger>
-      <DropdownMenuContent
-        asChild
-        align="end"
-        className="order static w-screen rounded-lg bg-white p-0 text-right md:static md:w-[180px]"
-      >
-        <div className="order fixed bottom-0 left-0 w-screen rounded-lg bg-white p-0 text-right md:static md:w-[180px]">
-          <DropdownMenuItem className="border-b px-6 py-3">
-            {!assetIsAvailable ? (
+      <figure>
+        {open && (
+          <style
```

Comment:
Okey so I don't like this at all so I checked the code to see why what we were trying to do was not working. 
You mentioned that this code:

```css
@media (max-width: 640px) {
  [data-radix-popper-content-wrapper] {
    transform: none !important;
  }
}
```

Is breaking the dialog we use for custody that Jurre mentioned. I did check it and thats not the case at all. Its just that the modal doesnt have a max-width so on mobile it was overflowing.

So please do the following:
 - [ ] restore the css code in `asset.css`
 - [ ] in `custom-modal.css` add `max-w-full` to the dialog class.

This will resolve the issue.

Reply:
@evonloch so I checked it further based on your comment on github and you are right. Was my mistake I was mistaken what the error referred to. So honestly I dont like the solution with the inline styles but at the moment due to how radix works I cannot find another easy solution so we can leave it like that. So you can ignore my previous comment and please do the following:
 - [ ] Not sure why you added the `<figure>`. I think that could be removed. If its needed for some specific reason, please let me know
 - [ ] Can you still do this: in `custom-modal.css` add `max-w-full` to the dialog class.
 - [ ] Please add a comment in the code where we have the style explaining why it works like this with a link to this PR, so when I see it in 1 year and I am wondering "What is this?" I can easily view the PR it was related to.

Once this is done, we can merge and close the issue.

Reply:
fixed